lib/icedfrisby.js

```diff
@@ -654,11 +654,16 @@ class Frisby {
     header = (header + "").toLowerCase()
     this.current.expects.push(function () {
       if (typeof self.current.response.headers[header] !== "undefined") {
-        chai.expect(self.current.response.headers[header].toLowerCase()).to
-          .contain(content.toLowerCase())
+        if(self.current.response.headers[header] instanceof Array){
+          chai.expect(self.current.response.headers[header]).to
+            .include.something.that.satisfies(thisHeader => thisHeader.toLowerCase().includes(content.toLowerCase())) // Ugly workaround for https://github.com/chaijs/chai-things/issues/42
```

Comment:
chaijs/chai-things#42 is about matching boolean values. How does it apply here?

Reply:
Same bug applies to `...include.something.that.contains(content)`, and same workaround resolves.
Ideally, I'd resolve at source, but one step at a time :)

Reply:
Ah, gotcha. Could you add a note to that effect in your comment, so it's clear to a future reader?

Reply:
Updated comment